kedro_mlflow/mlflow/kedro_pipeline_model.py

```diff
@@ -14,6 +14,8 @@ def __init__(self, pipeline_ml: PipelineML, catalog: DataCatalog):
         self.pipeline_ml = pipeline_ml
         self.initial_catalog = pipeline_ml.extract_pipeline_catalog(catalog)
         self.loaded_catalog = DataCatalog()
+        # we have the guarantee that there is only one output in inference
+        self.output_name = list(pipeline_ml.inference.outputs())[0]
```

Comment:
Since we have the guarantee that there is only one output in inference, can we just unpack blindly the `run_outputs` without creating a new class attribute "output_name" ? Do you foresee a future use of this attribute ?
If not, we can just unpack the `run_outputs` dict : `predictions, = run_outputs.values()`

Reply:
I was not sure if I should do that or do the way I did. I have a confused feeling that we may need in the future to access the `output_name` (e.g. for schema validation or a more declarative API), so I think it may be a good idea to have it as an attribute (and it does not add lot of overhead). So I think we can let it "as is" for now. Do you agree?

Reply:
ok ! let's keep the output_name